package/MDAnalysis/analysis/rms.py

```diff
@@ -293,49 +313,63 @@ class RMSD(AnalysisBase):
 
         1. any valid selection string for
            :meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms` that
-           produces identical selections in *mobile* and *reference*; or
+           produces identical selections in `atomgroup` and `reference`; or
+
+        2. a dictionary ``{'mobile': sel1, 'reference': sel2}`` where *sel1*
+           and *sel2* are valid selection strings that are applied to
+           `atomgroup` and `reference` respectively (the
+           :func:`MDAnalysis.analysis.align.fasta2select` function returns such
+           a dictionary based on a ClustalW_ or STAMP_ sequence alignment); or
 
-        2. a dictionary ``{'mobile':sel1, 'reference':sel2}`` (the
-           :func:`MDAnalysis.analysis.align.fasta2select` function returns
-           such a dictionary based on a ClustalW_ or STAMP_ sequence
-           alignment); or
         3. a tuple ``(sel1, sel2)``
 
-        When using 2. or 3. with *sel1* and *sel2* then these selections
-        can also each be a list of selection strings (to generate a
-        AtomGroup with defined atom order as described under
-        :ref:`ordered-selections-label`).
+        When using 2. or 3. with *sel1* and *sel2* then these selection strings
+        are applied to `atomgroup` and `reference` respectively and should
+        generate *groups of equivalent atoms*.  *sel1* and *sel2* can each also
+        be a *list of selection strings* to generate a
+        :class:`~MDAnalysis.core.groups.AtomGroup` with defined atom order as
+        described under :ref:`ordered-selections-label`).
+
     groupselections : list (optional)
-        A list of selections as described for `select` Each selection
+        A list of selections as described for `select`. Each selection
         describes additional RMSDs to be computed *after the structures
-        have be superpositioned* according to `select`. The output contains
+        have been superpositioned* according to `select`. The output contains
         one additional column for each selection. [``None``]
 
         .. Note:: Experimental feature. Only limited error checking
                   implemented.
 
-    filename : str, optional
-        write RSMD into file file :meth:`RMSD.save`
+    start : int (optional)
+        starting frame, default None becomes 0.
+    stop : int (optional)
+        Frame index to stop analysis. Default: None becomes
+        n_frames. Iteration stops *before* this frame number,
+        which means that the trajectory would be read until the end.
+    step : int (optional)
+        step between frames, default ``None`` becomes 1.
+    filename : str (optional)
+        write RMSD into file with :meth:`RMSD.save`
     mass_weighted : bool (deprecated)
          do a mass-weighted RMSD fit
-    weights : str/array_like (optional)
-         choose weights. If 'str' uses masses as weights
+    weights : {"mass", None} or array_like (optional)
```

Comment:
Ah, that makes sense. I asked mostly because also here, in the docs of `process_selection`, you explicitly used the *or ... or* notation.